fulp_modules/features/species/beefmen/mob/beefman_organs.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/organ/internal/brain/beefman
+	name = "minced-meat brain"
+	desc = "The inexplicably functional brain of a beefman. Some parts seem beyond any dream of repair though..."
+
+/obj/item/organ/internal/brain/beefman/Insert(mob/living/carbon/C, special, drop_if_replaced, no_id_transfer)
```

Comment:
single letter var in MY fulpcode? it's more likely than you think!

Reply:
😭